data/json/items/comestibles/med.json

```diff
@@ -1334,6 +1334,27 @@
     "use_action": "WEAK_ANTIBIOTIC",
     "flags": "NPC_SAFE"
   },
+  {
+    "id": "pepto",
+    "type": "COMESTIBLE",
+    "comestible_type": "MED",
+    "name": "heartburn medicine",
+    "description": "Creamy pink heartburn syrup that soothes upset stomachs and quells vomitus urges; with a twist off cap that doubles as a dosage cup.  Repeated doses may cause drowsiness.",
```

Comment:
vomitus -> vomitous

The former means 'matter that has been vomited' while the latter means 'nauseating'.